code/parachain/runtime/picasso/src/xcmp.rs

```diff
@@ -93,11 +93,7 @@ pub type Barrier = (
 	AllowSubscriptionsFrom<ParentOrSiblings>,
 	AllowTopLevelPaidExecutionFrom<Everything>,
 	TakeWeightCredit,
-	WithComputedOrigin<
-		AllowTopLevelPaidExecutionFrom<orml_xcm_builder_kusama::TinkernetMultisigMultiLocation>,
-		UniversalLocation,
-		ConstU32<8>,
-	>,
+	WithComputedOrigin<AllowTopLevelPaidExecutionFrom<Everything>, UniversalLocation, ConstU32<8>>,
```

Comment:
I concerned that prev it allowed the TopLevelPaidExecutionFrom only from TinkernetMultisigMultiLocation
but now you replaced to Everything,
Seems a big changes..

Reply:
I changed to `Everything` because it is considered safe to allow paid execution, as it requires the execution to be paid for in XCM fees + transaction fees. The [relay chain](https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/xcm_config.rs#L167) and the [system chains](https://github.com/polkadot-fellows/runtimes/blob/main/system-parachains/asset-hubs/asset-hub-kusama/src/xcm_config.rs#L452) both use the same barrier.

Would you prefer I keep this constrained to Tinkernet origins?